code/modules/mob/living/carbon/human/human_update_icons.dm

```diff
@@ -141,18 +141,20 @@ There are several things that need to be remembered:
 	update_mutant_bodyparts()
 
 /mob/living/carbon/human/update_worn_id()
-	remove_overlay(ID_LAYER)
+	//remove_overlay(ID_CARD_LAYER) //SKYRAT EDIT REMOVAL - PDA-B_GONE
 
 	if(client && hud_used)
 		var/atom/movable/screen/inventory/inv = hud_used.inv_slots[TOBITSHIFT(ITEM_SLOT_ID) + 1]
 		inv.update_icon()
 
-	var/mutable_appearance/id_overlay = overlays_standing[ID_LAYER]
+	//var/mutable_appearance/id_overlay = overlays_standing[ID_LAYER] //SKRAY EDIT REMOVAL - PDA B-GONE
 
 	if(wear_id)
 		var/obj/item/worn_item = wear_id
 		update_hud_id(worn_item)
-		var/icon_file
+		//var/icon_file SKYRAT EDIT REMOVAL - PDA-B-GONE
+//SKYRAT EDIT REMOVAL BEGIN - PDA-B-GONE
+/*
 
 		if(!icon_exists(icon_file, RESOLVE_ICON_STATE(worn_item)))
 			icon_file = 'icons/mob/clothing/id.dmi'
```

Comment:
The only line that actually needs commenting should be this one.

Reply:
There's a mirror coming from upstream that's going to change these lines, I'll merge it probably today, and then it should be as simple as changing one lines, really.

Reply:
![57inygvse5n51](https://user-images.githubusercontent.com/62520989/212411897-fe91399d-2897-4e83-ad0b-47bef5759f98.png)

Reply:
![image](https://user-images.githubusercontent.com/62520989/212411965-72213588-4cc9-41ae-9b7b-e17ddfd169ac.png)